app/src/main/res/values-ca/strings.xml

```diff
@@ -12,7 +12,7 @@
     <!-- Placeholder text shown in the search bar before a user enters text -->
     <string name="search_hint">Escriviu una cerca o adreça</string>
     <!-- No Open Tabs Message Header -->
-    <string name="no_open_tabs_header">No hi ha cap pestanya oberta</string>
+    <string name="no_open_tabs_header_2">No hi ha cap pestanya oberta</string>
```

Comment:
Not 100% sure, but I _think_ renaming this field in all these languages might prevent it from being put in queue for translation.  If you know this not to be the case, feel free to ignore.  Otherwise, would you mind removing `no_open_tabs_header_2` from all `strings.xml` files other than `values/strings.xml`?

Reply:
Thank you for the info @Baron-Severin!  When I make string changes on the DevTools team, we're required to create new keys, so I was assuming the same pattern would be required here.

Reply:
To clarify, creating a new key in `values/strings.xml` was definitely correct.  The process I went through on another Android team was to make a new string in that file without updating any of `values-ca`, `values-de`, etc.  I'm not sure how our a11y scripts work, but I worry that having existing translations will keep this one from being added to a translation queue somewhere.

But again, this is all conjecture on my part.  Hopefully someone with a better understanding of the system will jump in and correct me.

Reply:
*facepalm* You're right, I'll update this ASAP

Reply:
All good!  Intellij can be really overly helpful at times, which makes this an easy mistake to make.